zgrab2/multiple.go

```diff
@@ -0,0 +1,31 @@
+package zgrab2
+
+import (
+	"errors"
+	"os"
+
+	"github.com/ajholland/zflags"
+)
+
+type MultipleConfig struct {
+	ConfigFileName  string `short:"c" long:"config-file" default:"-" description:"Config filename, use - for stdin"`
+	ContinueOnError bool   `long:"continue-on-error" description:"If proceeding protocols error, do not run following protocols (default: true)"`
+}
+
+// Validates the options sent to MultipleConfig, and parses the configFile
+func (x *MultipleConfig) Validate(args []string) error {
+	if x.ConfigFileName == config.InputFileName {
```

Comment:
This seems like an excessive error check. Won't reading input just fail in this case?